rfc/003-remote-pipelines/README.md

```diff
@@ -0,0 +1,206 @@
+# Remote Pipelines
+
+- **Status**: WIP
+- **Created**: Sep 14, 2022
+- **ETA**: Oct 14, 2022
```

Comment:
I think it doesn't make much sense to start this discussion window before we at least have the basics of RFC-001 implemented. This will be so much easier to discuss once we've got the implementation experience.

Reply:
I'll update the ETA once RFC-001 is merged.